### DISCUSSION TITLE
Mollie payments platform does not show up in events.

### DISCUSSION THREAD

**Discussion by jfbethlehem** [alfio-event/alf.io#1195] [Q&A]
Hi,

using https://github.com/alfio-event/alf.io/discussions/1034 for Stripe, I got Mollie to connect (change stripe in the callback to mollie). I just cannot see Mollie get in the available payment options in the event:

![image](https://user-images.githubusercontent.com/7655441/218985890-160e8f82-be15-4fa5-9d4a-09c4c4f218da.png)

The odd thing is, Saferpay and Paypal are not even configured in the system yet do show up when not actively disabled. They have been disabled on a system-level (set to red) but not at an organisation level. There, the picker is set to grey.

Mollie Configuration is enabled in the system configuration:

![image](https://user-images.githubusercontent.com/7655441/218986376-2c922f51-473d-4ce6-9ecd-602552b30b3e.png)

And also in the organisation configuration:

![image](https://user-images.githubusercontent.com/7655441/218986672-14345989-2f1d-4cec-80ed-669710e93378.png)

Payment platform is enabled:

![image](https://user-images.githubusercontent.com/7655441/218987537-a042f45b-1e43-4f08-89f0-98bce4045f17.png)

Have I missed a step?

**Comment by jfbethlehem** [comment]
I found the reason. I had enabled Sofort and apparently Alfio crashes on that. Found that using the logs....Why does the payment method matter when the payment gateway does all the payment-work?
SEPA Direct Debit is also enabled but does not crash alfio.

**Comment by mihetah** [comment]
Yeah same for me when PayPal is selected in Mollie.
